sql/dql/basic-queries/where-2.md

```diff
@@ -31,65 +31,85 @@ aspects:
 ---
 ## Content
 
-### `AND`, `OR`, `NOT`, `BETWEEN`
+### `AND`, `OR`, `NOT`
 
-The predicates we use in `WHERE` can be conjoined using logical operators like `AND`, `OR` and `NOT`. They work as you would expect them to; you can include only the rows which satisfy both or either of two conditions using `AND` and `OR`, respectively. `NOT` negates the truth value of the predicate, so it now selects the rows that don't satisfy the condition.
+As you have seen in the previous insight, the predicates we use in `WHERE` can be conjoined using logical operators like `AND`, `OR` and `NOT`. They work as you would expect them to; you can include only the rows which satisfy both or either of two conditions using `AND` and `OR`, respectively. `NOT` negates the truth value of the predicate, so it now selects the rows that don't satisfy the condition.
```

Comment:
> They work as you would expect them to;

I think we can remove this

Reply:
maybe re-write this with bullet points to make it easier to follow:

These operators are used in combination with filtering conditions (e.g. `salary > 1000`) and can be used for:
- returning records for which two filter conditions are true at the same time with `AND`
- returning records for which any of two filter conditions are true with `OR`
- negating a filtering condition with `NOT`


might look for something else instead of "filtering condition", not sure how well this sounds
- in the case of `AND`,

Reply:
changed to:

As you have seen in the previous insight, the predicates we use in `WHERE` can be conjoined using logical operators like `AND`, `OR` and `NOT`. These operators are used in combination with filtering conditions (e.g. `salary > 1000`) and can be used for:

- returning records for which two (or more) conditions are true at the same time with `AND`
- returning records for which any of two (or more) conditions are true with `OR`
- negating a condition with `NOT`